railties/lib/rails/generators/rails/encrypted_secrets/encrypted_secrets_generator.rb

```diff
@@ -0,0 +1,66 @@
+require "rails/generators/base"
+require "rails/secrets"
+
+module Rails
+  module Generators
+    class EncryptedSecretsGenerator < Base
+      def add_secrets_key_file
+        unless File.exist?("config/secrets.yml.key") || File.exist?("config/secrets.yml.enc")
```

Comment:
If people have already opted in, then we may as well run `bin/rails secrets:edit` for them.

I'll tackle that post-merge though 👍